File: client/src/test/scala/stasis/test/specs/unit/client/ops/integration/mocks/MockLibrary.scala
package stasis.test.specs.unit.client.ops.integration.mocks

import scala.collection.concurrent.TrieMap

import org.apache.pekko.util.ByteString

class MockLibrary(
  val scheme: String,
  val entries: Map[String, ByteString]
) {
  val restored: TrieMap[String, ByteString] = TrieMap.empty
  val restoredAttributes: TrieMap[String, ByteString] = TrieMap.empty
}

File: client-android/lib/src/test/kotlin/stasis/test/client_android/lib/ops/integration/mocks/MockLibrary.kt
package stasis.test.client_android.lib.ops.integration.mocks

import okio.ByteString
import java.util.concurrent.ConcurrentHashMap

class MockLibrary(
    val scheme: String,
    val entries: Map<String, ByteString>
) {
    val restored: ConcurrentHashMap<String, ByteString> = ConcurrentHashMap()
    val restoredAttributes: ConcurrentHashMap<String, ByteString> = ConcurrentHashMap()
}
